src/index.tsx

```diff
@@ -1,11 +1,8 @@
-import React, { StrictMode } from 'react';
-import ReactDOM from 'react-dom';
+import React from 'react';
+import { createRoot } from 'react-dom/client';
 import './index.css';
 import App from './App';
 
-ReactDOM.render(
-  <StrictMode>
-    <App />
-  </StrictMode>,
-  document.getElementById('unlayer-editor-container'),
-);
+const container = document.getElementById('unlayer-editor-container');
+const root = createRoot(container!); // createRoot(container!) if you use TypeScript
+root.render(<App />);
```

Comment:
Is keeping the strict mode possible?

```suggestion
root.render(
  <StrictMode>
    <App />
  </StrictMode>);
```

Reply:
Yes, we can keep the strict mode.